docs/install.md

```diff
@@ -1182,21 +1219,47 @@ active volumes.
 
 #### DeploymentStatus
 
-A PMEM-CSI Deployment's `status` field is a `DeploymentStatus` object, which has
-a `phase` field. The phase of a Deployment is high-level summary of where the
-Deployment is in it's lifecycle.
+A PMEM-CSI Deployment's `status` field is a `DeploymentStatus` object, which
+carries the detailed state of the driver deployment. It comprises of [deployment
+conditions](#deployment-conditions), [driver component status](#driver-component-status),
+and a `phase` field. The phase of a Deployment is a high-level summary
+of where the Deployment is in its lifecycle.
 
 The possible `phase` values and their meaning are as below:
 
 | Value | Meaning |
 |---|---|
 | empty string | A new deployment. |
-| Initializing | All the direct sub-resources of the `Deployment` are created, but some indirect ones (like pods controlled by a daemon set) may still be missing. |
 | Running | The operator has determined that the driver is usable<sup>1</sup>.  |
-| Failed | For some reason the state of the `Deployment` failed and cannot be progressed<sup>2</sup>. |
+| Failed | For some reason the state of the `Deployment` failed and cannot be progressed. The failure reason is placed in the `DeploymentStatus.Reason` field. |
 
 <sup>1</sup> This check has not been implemented yet. Instead, the deployment goes straight to `Running` after creating sub-resources.
-<sup>2</sup> Failure reason is supposed to be carried by one of additional `DeploymentStatus` field, but not implemented yet.
+
+#### Deployment Conditions
+
+PMEM-CSI `DeploymentStatus` has an array of `conditions` through witch the
+PMEM-CSI Deployment has or has not passed. Below are the possible condition
+types and their meanings:
+
+| Condition type | Meaning |
+|---|---|
+| CertsReady | Driver certificates/secrets are available. |
+| CertsVerified | Verified that the provided certificates are valid. |
+| DriverDeployed | All the componentes required for the PMEM-CSI deployment has been deployed. |
+
+#### Driver component status
+
+PMEM-CSI `DeploymentStatus` has an array of `components` of type `DriverStatus`
+in which the operator records the brief driver(Controller and Node) components
```

Comment:
Space before opening bracket. Do we need the brackets here at all?

Why are controller and node capitalized?

I think "the brief driver components status" might be enough. What the components are is explained below.

Reply:
Change the text as suggested.